requirements.txt

```diff
@@ -4,4 +4,7 @@ pydocstyle==4.0.1
 pytest-flake8==1.0.4
 pytest-isort==0.3.1
 pytest==5.2.0
-pytest-mypy==0.4.2
\ No newline at end of file
+pytest-mypy==0.4.2
```

Comment:
These should be requirements-dev.txt (assuming we want to follow that pattern) as they will likely make any container (much) larger!

Reply:
If at some point we want to switch to poetry, we can replace this with just the .toml file.

Reply:
@jimoleary as discussed IRL, setup.py contains the production requirements, and requirements.txt contains the dev dependencies, so the containers will not install the requirements.txt requirements.

Reply:
agreed, this relates to my misunderstanding of the structure. Please ignore the original comment.